src/Veldrid/RgbaByte.cs

```diff
@@ -123,5 +133,70 @@ public override string ToString()
         {
             return string.Format("R:{0}, G:{1}, B:{2}, A:{3}", R, G, B, A);
         }
+
+        /// <summary>
+        /// Multiplies the channels of an RgbaByte by the given value.
+        /// </summary>
+        /// <param name="value">The source RgbaByte.</param>
+        /// <param name="scale">The value to multiply by.</param>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
We should either remove this or apply it to all the new methods, otherwise it's inconsistent.

Reply:
I think applying it to the other methods won't hurt.

|                    Method |     Mean |     Error |    StdDev | Scaled |
|-------------------------- |---------:|----------:|----------:|-------:|
|         RgbaByte_Multiply | 28.39 ns | 0.0952 ns | 0.0844 ns |   1.00 |
| RgbaByte_Multiply_Inlined | 20.21 ns | 0.1822 ns | 0.1521 ns |   0.71 |

``` ini

BenchmarkDotNet=v0.10.14, OS=Windows 10.0.17133.73 (1803/SpringCreatorsUpdate/Redstone4)
Intel Core i5-3210M CPU 2.50GHz (Ivy Bridge), 1 CPU, 4 logical and 2 physical cores
Frequency=2435886 Hz, Resolution=410.5282 ns, Timer=TSC
.NET Core SDK=2.1.300-preview1-008174
  [Host]     : .NET Core 2.0.6 (CoreCLR 4.6.26212.01, CoreFX 4.6.26212.01), 64bit RyuJIT
  DefaultJob : .NET Core 2.0.6 (CoreCLR 4.6.26212.01, CoreFX 4.6.26212.01), 64bit RyuJIT

```

Reply:
Sounds good to me.